doc/nrf/releases_and_maturity/migration/migration_guide_3.0.rst

```diff
@@ -207,32 +503,23 @@ Serial LTE Modem
 
    * Errors that were previously notified to the application with the ``LWM2M_CARRIER_ERROR_RUN`` event type have instead been added to :c:macro:`LWM2M_CARRIER_ERROR_CONFIGURATION`.
 
```

Comment:
For Wi-Fi,  please add:

Wi-Fi®
+------
+
+.. toggle::
+
+   * For samples using Wi-Fi features:
+
+      * nRF70 driver heap was part of the system shared heap ``CONFIG_HEAP_MEM_POOL_SIZE``. Now dedicated heaps have been defined for Wi-Fi driver control plane and data plane operations.
+        Default value of heap for control plane operations is 20000 bytes and for data plane operations, it is 130000. It is recommended to disable ``CONFIG_HEAP_MEM_POOL_IGNORE_MIN``,
+        and let system calculate the ``K_HEAP`` size. Any subsequent RAM overflow issues need
+        to be solved by fine tuning ``CONFIG_NRF_WIFI_CTRL_HEAP_SIZE`` and ``CONFIG_NRF_WIFI_DATA_HEAP_SIZE``.

Reply:
This will be added after the merge of https://github.com/nrfconnect/sdk-nrf/pull/21958.

Reply:
The change mentioned here has already been merged. So, this may not be dependent on merging of #21958

Reply:
OK. Added.